sphinx/directives/code.py

```diff
@@ -227,12 +227,13 @@ def read_file(self, filename: str, location: Tuple[str, int] = None) -> List[str
                     text = text.expandtabs(self.options['tab-width'])
 
                 return text.splitlines(True)
-        except OSError:
-            raise OSError(__('Include file %r not found or reading it failed') % filename)
-        except UnicodeError:
+        except OSError as e:
```

Comment:
We usually use `as exc` for such usage. So could you rename this to it?

Reply:
Fixed.